app/scripts/lib/auth-errors.js

```diff
@@ -145,6 +145,48 @@ define(function (require, exports, module) {
       errno: 135,
       message: t('Unable to deliver email')
     },
+    // Secondary Email errors
+    EMAIL_EXISTS: {
+      errno: 136,
+      message: t('This email was already verified by another user')
+    },
+    EMAIL_DELETE_PRIMARY: {
+      errno: 137,
+      message: t('Can not delete primary email')
+    },
+    SESSION_UNVERIFIED: {
+      errno: 138,
+      message: t('Unable to perform action, unverified session')
```

Comment:
also might not need this in the content-server, just like https://github.com/mozilla/fxa-content-server/pull/4996/files#r115769238 . At least we can remove the translation bit.